verify/test.sh

```diff
@@ -39,4 +39,4 @@ find_test_dirs() {
   )
 }
 
-go test $(find_test_dirs)
+GO111MODULE=off go test $(find_test_dirs)
```

Comment:
This isn't really necessary, right? We use golang1.11 and all the code is in the gopath inside the travis container, hence everything works as if GO111MODULE was set to off

Would you mind explaining more why we need this change? I don't see anything in the PR description

Reply:
This script is used also outside Travis, e.g. in the pre-push hook (it was also changed in this PR). I personally run those scripts on my machine, and with go1.13+ they stopped to work as modules are on by default. 

From Travis perspective, this shouldn't change anything.